Mathlib/Dynamics/Newton.lean

```diff
@@ -103,11 +103,11 @@ unit) then we may write `x` as a sum `x = n + r` where `n` is nilpotent and `r`
 Moreover, `n` and `r` are unique.
 
 This can be used to prove the Jordan-Chevalley decomposition of linear endomorphims. -/
-theorem exists_unique_nilpotent_sub_and_aeval_eq_zero
+theorem existsUnique_nilpotent_sub_and_aeval_eq_zero
```

Comment:
This one is missing a deprecation.

Reply:
Aah you're right, thanks so much for catching this!